### ISSUE TITLE
npm and travis badges don't work on scoped modules

### ISSUE THREAD

**Issue by nikoloza** [vigour-io/doc-badges#7]
Hey @shawninder :) disturbing today with many issues :)

I have two problems:
- [x] I want to be able to support badges in other `md` files as well (I think now only `README`) is supported. The great example why we may use it is this file: https://github.com/vigour-io/play-example/edit/master/COMPONENT.md
- [ ] NPM and Travis badges for scoped modules don't work. Usually, you can make [travis working](https://github.com/vigour-io/play-icon/blob/fa6aac3d8496cfd6712fe3eb1de5fee20d6dffb1/README.md), but not sure if you can do NPM badge for scoped (and **private**) packages with Fury or Shields.io.

Looking forward to hear from you about any issue and thanks ;)

**Comment by shawninder** [comment]
vdoc works on all `.md` files so that first issue is already solved :)
I will therefore rename this issue to represent only the second issue you mention and I'll tackle it when I'm done with my other priorities.

**Comment by nikoloza** [comment]
@shawninder cool thanks :)

So, it's like `vdoc file.md` right?

**Comment by shawninder** [comment]
yeah or `vdoc dirname` or just `vdoc` which is the same as `vdoc .`
